packages/admin/admin-rte/src/core/Controls/ControlButton.tsx

```diff
@@ -11,22 +11,9 @@ export interface IProps {
     onButtonClick?: (e: React.MouseEvent) => void;
     icon?: (props: SvgIconProps) => JSX.Element | null;
     children?: React.ReactNode;
-
-    /** @deprecated use icon instead */
-    Icon?: (props: SvgIconProps) => JSX.Element | null;
 }
 
-function ControlButton({
-    disabled = false,
-    selected = false,
-    onButtonClick,
-    icon,
-    children,
-    Icon: deprecatedIcon,
-    classes,
-}: IProps & WithStyles<typeof styles>) {
-    const Icon = icon || deprecatedIcon;
-
+function ControlButton({ disabled = false, selected = false, onButtonClick, icon: Icon, children, classes }: IProps & WithStyles<typeof styles>) {
```

Comment:
Why do we use `icon` lowercase here when we pass a component? Shouldn't it be uppercase for a component and lowercase for an element?